From 2a630343be0a57675ef801de12a35a32fe68306e Mon Sep 17 00:00:00 2001
From: Venkateshwar Rao Gannavarapu <venkateshwar.rao.gannavarapu@xilinx.com>
Date: Thu, 20 Aug 2020 14:35:45 -0700
Subject: [PATCH 1524/1851] dt-bindings: display: xlnx: sdi: Add gpio reset

commit 234b73ebb0d425527106412a67a71d65e236cf49 from
https://github.com/Xilinx/linux-xlnx.git

Add gpio reset entry for GT and PICXO cores of the subsystem.

Signed-off-by: Venkateshwar Rao Gannavarapu <venkateshwar.rao.gannavarapu@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt
index 701fef456765..a6f8640a3a2c 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt
@@ -16,6 +16,7 @@ Required properties:
  - clocks: List of phandles to AXI Lite, Video and SDI Tx Clock
  - clock-names: Must contain "s_axi_aclk", "video_in_clk" and "sdi_tx_clk"
    in same order as clocks listed in clocks property.
+ - phy-reset-gpio: Specifier for a gpio that asserts GT phy
 
 Optional properties:
  - xlnx,vpss: vpss phandle
@@ -23,6 +24,7 @@ Optional properties:
  - xlnx,tx-insert-c-str-st352: Insert ST352 payload in Chroma stream.
  - interrupt-names: Should be "sdi_tx_irq". This is only required when
    multiple interrupts are connected in the hardware design.
+ - picxo-reset-gpio: Specifier for a gpio that asserts picxo core.
 
 Example:
 
@@ -37,6 +39,7 @@ Example:
 		xlnx,vpss = <&v_proc_ss_0>;
 		clock-names = "s_axi_aclk", "video_in_clk", "sdi_tx_clk";
 		clocks = <&misc_clk_0>, <&misc_clk_1>, <&misc_clk_2>;
+		phy-reset-gpio = <&axi_gpio_0 0 0 0>;
 		ports {
 			#address-cells = <1>;
 			#size-cells = <0>;
-- 
2.31.1

